utils.js

```diff
@@ -67,17 +68,19 @@ var utils = () => {
     container.style.visibility = "hidden";
     frame.style.visibility = "hidden";
   }
-  
-  function openWidget() {
+
+  function openWidget(onEvent) {
     var container = document.getElementById("mono-connect--widget-div");
     var loader = document.getElementById("mono-connect-app-loader");
     var frame = document.getElementById("mono-connect--frame-id");
     container.style.visibility = "visible";
     container.style.display = "flex";
     loader.style.display = "block";
 
-    setTimeout(() => { 
-      turnOnVisibility(); 
+    onEvent('OPENED', metadata({}))
```

Comment:
opened event has to be from connect, that way we are sure the widget was opened, also would passed reference if available

Reply:
There is no event for open being published to the `message` event. Is this feature still coming?